module/VuFind/src/VuFind/View/Helper/Root/Component.php

```diff
@@ -67,6 +77,7 @@ public function __invoke(string $name, $params = []): string
 
         ++$invocation;
         $params['_invocation'] = $invocation;
+        $params['_themeClass'] = 'vc-' . $this->themeInfo->getTheme() . '-' . $name;
```

Comment:
Based on the above, it looks like `$name` may contain slash characters; do those need to be normalized to make this a valid CSS class?

Also, where does the 'vc-' prefix come from? Does that mean VuFind® component? I don't object to the prefix, but just want to be sure I understand it. And maybe a comment would be in order to explain.